projects/packages/forms/src/blocks/field-image-choice/index.tsx

```diff
@@ -0,0 +1,56 @@
+/**
+ * External dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+import { group as icon } from '@wordpress/icons';
+/**
+ * Internal dependencies
+ */
+import defaultSettings from '../shared/settings';
+import edit from './edit';
+import save from './save';
+
+const name = 'field-image-choice';
+
+const settings = {
+	...defaultSettings,
+	title: __( 'Image choice field', 'jetpack-forms' ),
+	keywords: [
+		__( 'Image', 'jetpack-forms' ),
+		__( 'Image choice', 'jetpack-forms' ),
+		__( 'Image choice field', 'jetpack-forms' ),
```

Comment:
We probably don't need any of these;

"Image choice field" already covers all the words in the previous two, but it's also the field name which also uses as a search term.